storybook/index.android.js

```diff
@@ -0,0 +1,3 @@
+import StorybookUI from './storybook';
```

Comment:
why is there an android and iOS and index file with the same code? Isn't it possible to remove android and ios files

Reply:
Not needed indeed, I removed them.